### PR TITLE
Style guide updates for local name changes. Fixes #507.

### LINKED ISSUE #507: Make various additions to style guide based on decisions made on issues
Review issues and PRs for 10.0.0 to find all cases; especially local names issue #188.

### PR DESCRIPTION
**Pull request by rjyounes**

### CODE REVIEW

**Review by sa-bpelakh — APPROVED** [review]
Looks good, though I have to say the title casing of `Purpose of This Style Guide` threw me off for a moment.

**Comment by rjyounes** [comment]
@sa-bpelakh Indeed, I had to check our title case policy to get that right!

File: docs/gistStyleGuide.md [hunk 1]
```diff
@@ -45,28 +46,81 @@ Serialization
...
+| Object properties verb-initial | `isAbout`, not `about` |
+||`comesFromAgent`, not `fromAgent`|
+|| `isMemberOf`, not `memberOf` |
+|| `precedesDirectly`, not `directlyPrecedes` |
+|| `usesTimeZoneStandard`, not `timeZoneStandardUsed` |
+| Prefix "is" to "-ed" forms, both past participles and adjectives | `isGovernedBy`, not `governedBy` |
+|| `isCharacterizedBy`, not `characterizedBy` |
+| Prefer an ordinary verb to "hasX" or "isX", even in a pair of inverses | `follows`, not `isPrecededBy`, even when inverse `precedes` exists |
+| "At" rather than "on" for datetimes | `isRecordedAt`, not `isRecordedOn`. |
+| Present tense only with minimal exceptions when the meaning is inherently in the past | `isRenderedOn`, not `wasRenderedOn`|
```
[the remark this answers is not in the document]
Comment (uscholdm) [reply docs/gistStyleGuide.md:95]: Sorry a bit late, just do this some other time, if at all.

[the remark this answers is not in the document]
Comment (rjyounes) [reply docs/gistStyleGuide.md:95]: Good idea. I'll submit a new PR if you could just review it very quickly.

**Pull request merged**